### ISSUE TITLE
How can we change svg height and width?

### ISSUE THREAD

**Issue by elzodxon** [mkolsv/circle-progress.vue#1]
I want to customize component and change width and height.

**Comment by mkolsv** [comment]
Hi, @elzodxon !

I will add props for it in the next update.

Now you can change the size with CSS.
For example:

```
.circle-progress__svg {
    width: 500px;
    height: 500px;
}
```

**Comment by mkolsv** [comment]
Hi @elzodxon again!

I just released v3.1.0

So now you can pass prop `size` to set custom size of circle
